src/DistortableCollection.js

```diff
@@ -2,6 +2,7 @@ const arr = [];
 L.DistortableCollection = L.FeatureGroup.extend({
   options: {
     editable: true,
+    tooltipControl: document.createElement('button'),
```

Comment:
Hi @segun-codes -- I appreciate this toggling system here, and I know it's really challenging to dig into the guts of this library, and you've done a really really great job writing well-organized, easy to read code here. So congratulations. 

However, I am just thinking... weighing complexity vs. benefit in balance, I ultimately think it's not worth it to have a permanent button-management system within the core library code. I have to think of what else people might do with tooltips, if they'd work the same way, if this code would need maintenance in the distant future, and so on... and ultimately you really solved most of the problem of the tooltips potentially interfering with things by making tooltips default off. 

I'd like to ask you to keep the tooltip /creation/ code as it is in `src/DistortableImageOverlay.js`, but to remove the UI code from this file, and the button toggling the tooltips. I think it presents the user with another button which has a function not related to actual mapmaking, and I worry it's just not worth the added complexity, both for code, and for the user having to learn what the button is for. 

I hope you understand and agree -- and once more, you've done a stellar job at architecting this UI and managing it, and integrating code into a complex library. I just think the better judgement call is to keep our changes to the core library simple and concise. 

Thank you, Segun!!

Reply:
For now, can you save the button management code in an issue, and we can wait and see if any issues ever come up with the tooltips obscuring or blocking things, and we can make a decision about including the button code later? 

That will also keep this PR a bit narrower in scope and still achieve our main goal, but give us flexibility to respond later if we find the tooltips need more work. How does that sound? Thank you!

Reply:
Sounds okay, since it's for experimental purposes for the time being.

Reply:
> For now, can you save the button management code in an issue, and we can wait and see if any issues ever come up with the tooltips obscuring or blocking things, and we can make a decision about including the button code later?
> 
> That will also keep this PR a bit narrower in scope and still achieve our main goal, but give us flexibility to respond later if we find the tooltips need more work. How does that sound? Thank you!

@jywarren, I have reverted to the earlier implementation with some improvements consistent with your advice. Kindly review and perhaps, consider for merging too. Thank you for the support.